components/director/internal/domain/formation/fixtures_test.go

```diff
@@ -217,6 +218,12 @@ func fixApplicationModelWithoutTemplate(applicationID string) *model.Application
 	return appModel
 }
 
+func fixApplicationWithType(applicationType string) *model.Application {
```

Comment:
Don't see the param applicationType being used here

Reply:
Forgot to remove and adapt it, it's only used in one test anyway. I will remove it.